### PR TITLE
Migrate diagram in ERTS guides to mermaid.js

### PR DESCRIPTION
**Pull request by Benjamin-Philip**
This commit migrates the EMPD Requests Diagram to mermaid.js, which is
the sole diagram in ERTS guides.

### CODE REVIEW

**Comment by Benjamin-Philip** [comment]
Ideally I would have done all the diagrams in ERTS at one go, but flamegraphs are not supported in Mermaid (for the BeamAsm docs). I also chose to leave the diagrams in the GC docs as I felt it would take a long time to port.

File: erts/doc/guides/erl_dist_protocol.md [hunk 1]
```diff
@@ -57,7 +57,43 @@ The integers in all multibyte fields are in big-endian order.
...
+    participant EPMD
+
+    Note over EPMD: Register a Node
+    client ->> EPMD: ALIVE2_REQ
+    EPMD -->> client: ALIVE2_X_RESP
+
+    Note over EPMD: Unregister a Node
+    client ->> EPMD: ALIVE_CLOSE_REQ
+
+    Note over client: Get the Distribution Port of Another Node
```
Comment (garazdawi) [inline erts/doc/guides/erl_dist_protocol.md:76]: I wonder if there is any better way to display these notes. Maybe it is better to split into multiple diagrams? Or convert this summary to a billet list and move the diagrams into each respective section?

Comment (Benjamin-Philip) [reply erts/doc/guides/erl_dist_protocol.md:76]: I was using the notes to segregate the different requests. A bullet list, and separate diagrams sound good to me. I feel that this diagrams works well as an overview or summary diagram as originally intended. So what I would do is keep the diagram, and add a bullet list, as well as separate diagrams.

**Comment by garazdawi** [comment]
thanks!

**Pull request merged**